include/nbl/builtin/hlsl/fft/common/twiddles_impl.hlsl

```diff
@@ -0,0 +1,291 @@
+#ifndef float_t
+#error Define float_t before including 
+#endif
+#ifndef TYPED_NUMBER
+#error Define TYPED_NUMBER before including 
+#endif
+
+#ifndef LOW_TWIDDLE_BITS
+#error Define LOW_TWIDDLE_BITS before including 
+#endif
+#ifndef MID_TWIDDLE_BITS
+#error Define MID_TWIDDLE_BITS before including 
+#endif
+#ifndef HIGH_TWIDDLE_BITS
+#error Define HIGH_TWIDDLE_BITS before including 
+#endif
+
+#define NAMESPACE_NAME NBL_CONCATENATE(twiddles_, float_t) 
+
+namespace NAMESPACE_NAME {
+    
+    // -------------------------------------------- 64th roots of unity --------------------------------------------------------
+
+    NBL_CONSTEXPR float_t low_real[1u << LOW_TWIDDLE_BITS] = {
+        TYPED_NUMBER(1),
```

Comment:
I'm pretty sure that compiler will just choke with
```
too many elements in vector initialization (expected X element, have Y)
```
when `1<<LOW_TWIDDLE_BITS` is less than the length of your array, as this is what godbolt gives on
```
const static float arr[1] = {6.f,9.f}
```